jekyll/_cci2/api_v2_introduction.md

```diff
@@ -0,0 +1,226 @@
+
+For the project type you can use `github` or `bitbucket` as well as the shorter forms `gh` or `bb`, which are now supported in API v2. The `organization` is your username or organization name in your version control system.
+
+With API v2, CircleCI has introduced a string representation of the triplet called the `project_slug`, and takes the following form: 
+
+`<project_type>/<org_name>/<repo_name>`
+
+The `project_slug` is included in the payload when pulling information about a Project as well as when looking up a pipeline or workflow by ID. The `project_slug` can then be used to get information about the project. It is possible in the future CircleCI may change the shape of a `project_slug`, but in all cases it would be usable as a human-readable identifier for a given project.
+
+## Using 'when' in Workflows
+
+With this new version of the API, you may use a `when` clause (the inverse clause is also supported) under a workflow declaration with a boolean value to determine whether or not to run that workflow.
+
+The most common use of `when` in API v2 is to use a pipeline parameter as the value, allowing an API trigger to pass that parameter to determine which workflows to run.
+
+The example below shows an example configuration using two different pipeline parameters. You may use one of the parameters to drive whether a particular workflow, while you may use the other parameter determines whether a particular step will run.
+
+```yaml
+version: 2.1
+
+parameters:
+  run_integration_tests:
+    type: boolean
+    default: false
+  deploy:
+    type: boolean
+    default: false
+
+workflows:
+  version: 2
+  integration_tests:
+    when: << pipeline.parameters.run_integration_tests >>
+    jobs:
+      - mytestjob
+      - when:
+          condition: << pipeline.parameters.deploy >>
+          steps:
+            - deploy
+
+jobs:
+...
+```
+ 
+This example prevents the workflow integration_tests from being triggered unless the tests were invoked explicitly when the pipeline is triggered with the following in the `POST` body:
+
+```
+{
+    "parameters": {
+        "run_integration_tests": true
+    }
+}
+```
+
+The `when` key actually accepts any boolean value, not just pipeline parameters. Although pipeline parameters will be the primary use of this feature until CircleCI implements other uses, `when` also has an inverse clause called `unless`, which inverts truthiness of the condition.
+
+## Changes In Endpoints
+
+The CircleCI API v2 release includes several new endpoints that you can use in your jobs, as well as some endpoints that have been deprecated. The sections below list the endpoints added for this release, in addition to the endpoints that have been removed.
```

Comment:
Let's not say "you can use in your jobs" - that implies you can't use this API outside of jobs.

Reply:
fixed